### PR TITLE
LPS-152935 Use always trash icon and "delete" label for removal action regardless of having recycle bin or not

### PR DESCRIPTION
**Pull request by victorg1991**
Hello good people of Lima 👋 😄 

There is an initiative from design to normalize the use of "delete" action regardless of the presence of recycle bin or not :) 

This PR changes all the labels to "delete" and all the icons to "trash"

Any questions/comments are welcome : D

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
Quarantined test MentionsBlogs#CanViewMentionViaBlogsEntryCommentAsGuest under https://github.com/brianchandotcom/liferay-portal/pull/117049

**Comment by victorg1991** [comment]
Thanks @Hanlf :)

**Comment by Hanlf** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/logic/UIItemsBuilder.java [hunk 1]
```diff
@@ -701,7 +701,7 @@ public void addMoveToTheRecycleBinToolbarItem(
 		_addJavaScriptUIItem(
 			new JavaScriptToolbarItem(), toolbarItems,
 			DLUIItemKeys.MOVE_TO_THE_RECYCLE_BIN,
-			LanguageUtil.get(_resourceBundle, "move-to-recycle-bin"),
+			LanguageUtil.get(_resourceBundle, "delete"),
```
Comment (ambrinchaudhary) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/logic/UIItemsBuilder.java:704]: Here we may also want to rename the method ( _addMoveToTheRecycleBinToolbarItem_ to  _addDeleteToolbarItem_).

Comment (victorg1991) [reply modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/logic/UIItemsBuilder.java:704]: I believe the meaning is the same, we are only changing the label. This is still the Move to the recycle bin action, isn't it?

**Review by ambrinchaudhary — APPROVED** [review]

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/logic/UIItemsBuilder.java (hunk 1 above)
Comment (ambrinchaudhary) [reply modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/logic/UIItemsBuilder.java:704]: Ok, I see your point. Then we can leave the method name as it is... thanks!

**Review by boton — APPROVED** [review]

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**